rosidl_generator_cpp/resource/msg__struct.hpp.template

```diff
@@ -107,8 +107,8 @@ for field in spec.fields:
 
   // field types and members
 @[for field in spec.fields]@
-  typedef @(msg_type_to_cpp(field.type))
-    _@(field.name)_type;
+  using _@(field.name)_type =
+    @(msg_type_to_cpp(field.type));
```

Comment:
I don't mind, I thought our style code didn't say anything about indenting twice after wrapping.

Just fixed it, thanks!